src/firefly/js/visualize/ui/FitsHeaderView.jsx

```diff
@@ -0,0 +1,183 @@
+			   </div>
+			   <div style={helpIdStyle}>
+				  <HelpIcon helpid={'visualization.fitsDownloadOptions'} />
+			   </div>
+            </div>
+		</div>
+	);
+
+};
+function showFitsHeaderPopup(plot, tableId, fitsHeaderInfo) {
+
+	var popupId=popupIdRoot+'_'+tableId;
+
+	const popTitle = 'FITS Header:'+plot.title;
+	var popup = (<PopupPanel title={popTitle}>
+			{popupForm(plot, fitsHeaderInfo, popupId)}
+		</PopupPanel>
+
+	);
+
+	DialogRootContainer.defineDialog(popupId, popup);
+	dispatchShowDialog(popupId);
+}
+
+function render3BandFitsHeaders(plot, fitsHeaderInfo) {
+
+	return (
+		<div style = {tabStyle} >
+
+				<Tabs   defaultSelected={0}>
+					<Tab  name='RED'>
+						<div style = { mTableStyle} >
+                            {renderSingleBandFitsHeader(plot, Band.RED, fitsHeaderInfo)}
+                       </div>
+					</Tab>
+
+					<Tab name= 'GREEN' >
+						<div style = { mTableStyle} >
+						    {renderSingleBandFitsHeader(plot, Band.GREEN, fitsHeaderInfo)}
+						</div>
+					</Tab>
+					<Tab name='BLUE'>
+						<div style = { mTableStyle} >
+						    {renderSingleBandFitsHeader(plot, Band.BLUE, fitsHeaderInfo)}
+						</div>
+					</Tab>
+
+
+			</Tabs>
+
+		</div>
+
+
+	);
+
+}
+
+function renderSingleBandFitsHeader(plot, band,fitsHeaderInfo){
+
+
+	const  tableModel = JSON.parse(fitsHeaderInfo[band]);
+
+	const tableData =tableModel.tableData;
+
+	const  data = tableData.data;
+	const  columns = tableData.columns;
+	const  pt = plot.projection.getPixelScaleArcSec();
+	const  pixelSize = pt.toFixed(2)+'"';
+	const  meta = tableModel.tableMeta;
+	const  fileSize = Math.round(meta.fileSize/1000.0) +'K';
+
+   return (
+
+		  <div style = {tabStyle}>
+			 <div style={titleStyle}>  <div style= { {display:'inline-block', width: 100} }>PixelSize:{pixelSize} </div>
+				 <div  style= { {display:'inline-block', width: 100} }>  FileSize:{fileSize}</div>
+			 </div>
+			 <div  style={tableStyle}>
+			    <BasicTable
+				columns={columns}
+				data={data}
+				height='calc(100% - 42px)'
+		     	/>
+			 </div>
+		  </div>
+
+	);
+
+}
+
+/**
+ * This function calls the server function to calculate the FitsHeader information
+ * @param plot
+ * @param tableId
+ */
+
+function showFitsHeaderData(plot, tableId){
+
+
+	callGetFitsHeaderInfo(plot.plotState, tableId)
+		.then( (result) => {
+
+			console.log(result);
+
+			showFitsHeaderPopup(plot, tableId, result);
+		})
+		.catch ( (e) => {
+			console.log(`fitsHeader error: ${plot.plotId}`, e);
+
+		});
+}
+
+/**
+ *
+ *  This function will return the popup component.  As React conversion, the CamelCase is used.
+ * @param plotView
+ * @constructor
+ */
+export function  FitsHeaderView(plotView){
+
+	var plot= primePlot(plotView);
+	if  (!plot)  return;
+
+	var str=plot.title.replace(/\s/g, '');
+	var tableId= str.replace(/[^a-zA-Z0-9]/g, '_');
+
+	showFitsHeaderData(plot, tableId);
+
+
+}
+FitsHeaderView.propTypes= {
+	plotView: PropTypes.object.isRequired
```

Comment:
since this is just a function and not a react component it does not need propTypes.